### PR TITLE
sleep if invalid ulid

### LINKED ISSUE #14: CLI should gracefully handle MonotonicError
Currently it just calls `.unwrap()`.

### PR DESCRIPTION
**Pull request by camerondavison**
fixes #14

### CODE REVIEW

File: cli/src/bin/ulid.rs [hunk 1]
```diff
@@ -29,11 +30,24 @@ fn main() {
...
-            writeln!(&mut locked, "{}", gen.generate().unwrap()).unwrap();
+        let mut i = 0;
+        while i < count {
+            match gen.generate() {
+                Ok(ulid) => {
+                    writeln!(&mut locked, "{}", ulid).unwrap();
+                    i += 1;
+                }
+                Err(_) => {
+                    writeln!(&mut err_locked, "unable to create ulid").unwrap();
```
Comment (dylanhart) [inline cli/src/bin/ulid.rs]: Can this be something with more explanation like "Failed to create new ulid due to overflow, sleeping 1 ms"

File: cli/src/bin/ulid.rs [hunk 2]
```diff
@@ -29,11 +30,24 @@ fn main() {
...
+    let mut err_locked = stderr.lock();
     if monotonic {
         let mut gen = Generator::new();
-        for _ in 0..count {
-            writeln!(&mut locked, "{}", gen.generate().unwrap()).unwrap();
+        let mut i = 0;
+        while i < count {
+            match gen.generate() {
+                Ok(ulid) => {
+                    writeln!(&mut locked, "{}", ulid).unwrap();
```
Comment (dylanhart) [inline cli/src/bin/ulid.rs]: Fixing the these unwraps would be great too!

Comment (camerondavison) [reply cli/src/bin/ulid.rs]: what do you think a good thing to do is when this fails? try and log to stderr and exit?

Comment (dylanhart) [reply cli/src/bin/ulid.rs]: Yeah, that's what stdlib does.

**Pull request merged**